grid/project-management/projects/maps/__snapshots__/asana.test.ts.snap

```diff
@@ -1,12 +1,14 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`project-management/projects/asana ListProject the usecase without any parameters should list all projects 1`] = `
-Ok {
-  "value": Array [
-    Object {
-      "id": "1202159846567214",
-      "name": "Customer implementation",
-    },
-  ],
+Err {
+  "error": "Security values for security scheme not found: personalAccesstoken
```

Comment:
Watch out, this ended up in error, I asssume this is not intentional (and yeah, testing library doesn't expose error as a failure, it sucks, I know).

Reply:
😬  it isn't